assignment/assignment-tool/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -31,3 +31,27 @@ ASN_SVS.undoCancel = function()
 	submitPanel.style.display = "block";
 	confirmationDialogue.style.display = "none";
 };
+
+$(document).ready(function() {
+	$(document).click(function (event) {
+		// If an attachment has been added and the user tries to click away from the page show a warning. 
+		if ($("#attachments").attr("href").length != 0 && $(event.target).is('a, a *')) {
```

Comment:
Does the remove link still work once you have attachments as doesn't this capture that event?